lms/djangoapps/instructor_task/tasks.py

```diff
@@ -81,6 +82,19 @@ def rescore_problem(entry_id, xmodule_instance_args):
 
 
 @task(base=BaseInstructorTask)  # pylint: disable=not-callable
+def override_problem_score(entry_id, xmodule_instance_args):
+    """
+    Overrides a specific learner's score on a problem.
+    """
+    # Translators: This is a past-tense verb that is inserted into task progress messages as {action}.
+    action_name = ugettext_noop('overridden')
```

Comment:
@srpearce here's a user-facing term. It gets inserted into a notification along the lines of "Problem score successfully ___ for learner ___."